views/admin.functions.setup.options.php

```diff
@@ -650,6 +690,9 @@ function sailthru_setup_handler( $input ) {
 		}
 	}
 
+	// recaptcha settings
+	$output['google_recaptcha_site_key'] = isset( $input['google_recaptcha_site_key'] ) ? filter_var( $input['google_recaptcha_site_key'], FILTER_SANITIZE_STRING ) : '';
+	$output['google_recaptcha_secret'] = isset( $input['google_recaptcha_secret'] ) ? filter_var( $input['google_recaptcha_secret'], FILTER_SANITIZE_STRING ) : '';
```

Comment:
Non-blocking comment:
This might be easier to read without the use of ternary operators:
```
$output['google_recaptcha_site_key'] = '';
if ( isset( $input['google_recaptcha_site_key'] ) ) {
    $output['google_recaptcha_site_key'] = filter_var( $input['google_recaptcha_site_key'], FILTER_SANITIZE_STRING );
}

$output['google_recaptcha_secret'] = '';
if ( isset( $input['google_recaptcha_secret'] ) ) {
    $output['google_recaptcha_secret'] = filter_var( $input['google_recaptcha_secret'], FILTER_SANITIZE_STRING );
}
```